core/iwasm/libraries/libc-wasi/sandboxed-system-primitives/src/posix.c

```diff
@@ -323,37 +325,19 @@ __wasi_errno_t
 wasmtime_ssp_clock_res_get(__wasi_clockid_t clock_id,
                            __wasi_timestamp_t *resolution)
 {
-#ifdef BH_PLATFORM_WINDOWS
-    return __WASI_ENOSYS;
-#else
-    clockid_t nclock_id;
-    if (!convert_clockid(clock_id, &nclock_id))
-        return __WASI_EINVAL;
-    struct timespec ts;
-    if (clock_getres(nclock_id, &ts) < 0)
+    if (os_clock_res_get(clock_id, *resolution) != BHT_OK)
         return convert_errno(errno);
-    *resolution = convert_timespec(&ts);
-    return 0;
-#endif
+    return __WASI_ESUCCESS;
 }
 
 __wasi_errno_t
 wasmtime_ssp_clock_time_get(__wasi_clockid_t clock_id,
                             __wasi_timestamp_t precision,
-                            __wasi_timestamp_t *time)
-{
-#ifdef BH_PLATFORM_WINDOWS
-    return __WASI_ENOSYS;
-#else
-    clockid_t nclock_id;
-    if (!convert_clockid(clock_id, &nclock_id))
-        return __WASI_EINVAL;
-    struct timespec ts;
-    if (clock_gettime(nclock_id, &ts) < 0)
+                            __wasi_timestamp_t *time){
+    
+    if(os_clock_time_get(clock_id,precision,*time) !=BHT_OK);
```

Comment:
I think you don't mean to have the semicollon at the end
```suggestion
    if(os_clock_time_get(clock_id,precision,*time) !=BHT_OK)
```